source/docs/getting-started/running-a-benchtop/creating-benchtop-test-program-cpp-java.rst

```diff
@@ -39,7 +39,7 @@ Once all the above have been configured, click "Generate Project" and the robot
 Opening The New Project
 -----------------------
 
-After successfully creating your project, VS Code will give the option of opening the project as shown below. We can choose to do that now or later by typing Ctrl-O (Command+O on macOS) and select the folder where We saved our project.
+After successfully creating your project, VS Code will give the option of opening the project as shown below. We can choose to do that now or later by typing Ctrl-K (Command+O on macOS) and select the folder where we saved our project.
```

Comment:
Should it also be Command+K on macOS?

Reply:
Nope. Mac is able to select either a file or directory with a single dialog.